bigquery/test/queries.test.js

```diff
@@ -85,7 +85,7 @@ describe('bigquery:query', function () {
     });
 
     it('should handle error', function () {
-      var error = Error('syncQueryError');
```

Comment:
Should this be a global variable? (Or do we have to create a new one within each test?)